source/MRMesh/MRMeshDelone.h

```diff
@@ -1,10 +1,22 @@
 #pragma once
 
 #include "MRMeshFwd.h"
+#include "MRProgressCallback.h"
 
 namespace MR
 {
 
+constexpr int NoAngleChangeLimit = 10;
+
+struct DeloneSettings
+{
+    int numIters = 0;
```

Comment:
Please add comments to each field (they can be found below)
numIters = 1 by default